deployer/src/deployment/run.rs

```diff
@@ -513,30 +527,60 @@ mod tests {
         }
     }
 
-    fn get_runtime_manager() -> Arc<Mutex<RuntimeManager>> {
+    pub struct MockedLogger;
+
+    #[async_trait]
+    impl Logger for MockedLogger {
+        async fn store_logs(
+            &self,
+            _: Request<StoreLogsRequest>,
+        ) -> Result<Response<StoreLogsResponse>, Status> {
+            Ok(Response::new(StoreLogsResponse { success: true }))
+        }
+
+        async fn get_logs(
+            &self,
+            _: Request<LogsRequest>,
+        ) -> Result<Response<LogsResponse>, Status> {
+            Ok(Response::new(LogsResponse {
+                log_items: Vec::new(),
+            }))
+        }
+
+        type GetLogsStreamStream = ReceiverStream<Result<LogLine, Status>>;
+
+        async fn get_logs_stream(
+            &self,
+            _: Request<LogsRequest>,
+        ) -> Result<Response<Self::GetLogsStreamStream>, Status> {
+            let (_, rx) = mpsc::channel(1);
+            Ok(Response::new(ReceiverStream::new(rx)))
+        }
+    }
```

Comment:
Can we replace this with the one from `common-tests`?

Reply:
We don't have a Logger impl in common-tests, but we can move this there. It's only used in one place currently, though.

Reply:
I put the impl in common-tests.